### PR TITLE
Extended UI Features

### PR DESCRIPTION
**Pull request by aanu2021**

### CODE REVIEW

File: ts-frontend/src/components/Map.tsx [hunk 1]
```diff
@@ -156,6 +182,12 @@ export const Map = () => {
     closeConnection("202A", currentUserUUID);
   };
 
+  const DEFAULT_PROFILE_URL =
+    "https://cdn-icons-png.flaticon.com/128/3177/3177440.png";
+  const DEFAULT_MARKER_PIC =
+    "https://res.cloudinary.com/cantacloud2/image/upload/w_40,h_40,c_scale/v1714413297/hbjh5jqrguqtuvmfr7sl.png";
```
Comment (sanam2405) [inline ts-frontend/src/components/Map.tsx:188]: export/import these constant URIs to and from the `constants` directory. 
this should make the default URIs consistent across the codebase

File: ts-frontend/src/components/Map.tsx [hunk 2]
```diff
@@ -238,12 +347,86 @@ export const Map = () => {
             <GoogleMap
               mapContainerStyle={containerStyle}
               center={currentMapCenter}
-              zoom={12}
+              zoom={9}
             >
               <Marker
-                key="11111"
-                position={{ lat: adminLocation.lat, lng: adminLocation.lng }}
+                key={9999}
```
Comment (sanam2405) [inline ts-frontend/src/components/Map.tsx:353]: this key should be the `currentUserUUID`

File: ts-frontend/src/components/Map.tsx [hunk 3]
```diff
@@ -164,6 +196,28 @@ export const Map = () => {
...
+          } else {
+            setcurrUser(data.user);
+            localStorage.setItem("user", JSON.stringify(data.user));
+          }
+        })
+        .catch((err) => console.log(err));
+    }
+  };
+
   const updateDetails = () => {
```
Comment (sanam2405) [inline ts-frontend/src/components/Map.tsx:221]: this function is used to get the details of the the users that are queried  by the current client with the values taken from the distance, age slider and college, gender drop-down.

we also have a ` updateCurrentLocation` function that updates the currentUser (user with id: _currentUserUUID_) location and writes it to the Postgres through the ts-backend server.

these two function names are ambiguous so,
rename this  `updateDetails` function to `updateDetailsOfQueriedUsers` for better clarity

File: ts-frontend/src/components/Map.tsx (hunk 1 above)
Comment (aanu2021) [reply ts-frontend/src/components/Map.tsx:188]: done

File: ts-frontend/src/components/Map.tsx (hunk 2 above)
Comment (aanu2021) [reply ts-frontend/src/components/Map.tsx:353]: done

File: ts-frontend/src/components/Map.tsx (hunk 3 above)
Comment (aanu2021) [reply ts-frontend/src/components/Map.tsx:221]: done

**Review by sanam2405 — APPROVED** [review]
LGTM

**Pull request merged**